src/lib/isEmail.js

```diff
@@ -49,6 +50,17 @@ export default function isEmail(str, options) {
       should be done in normalizeEmail
     */
     user = user.toLowerCase();
+
+    // Removing sub-address from username before gmail validation
+    const username = user.split('+')[0];
+
+    if (!isByteLength(username, { min: 6, max: 30 })) {
```

Comment:
Do you have a source for the [6, 30] length restriction? and the alphanumeric char restriction?

Reply:
I can't find a link in google/gmail support except for G-suite users but both length and characters restrictions can be found in Gmail sign-up page

Reply:
K, confirmed:

<img width="404" alt="screen shot 2018-05-16 at 8 37 52 pm" src="https://user-images.githubusercontent.com/364903/40112343-0c7187d2-5949-11e8-9290-3a681add8930.png">

<img width="399" alt="screen shot 2018-05-16 at 8 37 42 pm" src="https://user-images.githubusercontent.com/364903/40112348-0e30bb06-5949-11e8-9f58-49aadd5eb670.png">